test/unit/config/AppConfigSpec.scala

```diff
@@ -143,6 +143,12 @@ class AppConfigSpec extends ControllerSpec with BeforeAndAfterEach {
       ) shouldBe "http://localhost:6753/vat-known-facts-control-list"
     }
 
+    "register for an EORI link takes user to ECC" in {
+      appConfig.eoriCommonComponentRegistrationFrontend(
+        atarService.code
+      ) shouldBe "http://localhost:6751/customs-registration-services/atar/register"
+    }
+
```

Comment:
Beat me to the comment - but can we have this be programmatic (it should be able to iterate through all codes and produce them appropriately) or an Arbitrary[Service].